css/index.css

```
@@ -0,0 +1,74 @@
+body {
+    background-color: darkcyan;
+    color: darkblue;
+    font-family: lato;
+    margin: auto;
+    padding-top: 30px;
+}
+
+#Header {
```

Quick tip: just as a convention, we usually name ids with all lowercase letters

```suggestion
#header {
```